nested-labvm/atd-docker/uilanding/src/uilanding.py

```diff
@@ -25,6 +26,21 @@
 
 ArBASE_PATH = '/opt/modules/'
 MODULE_FILE = ArBASE_PATH + 'modules.yaml'
+MENU_BASE_PATH = '/opt/menus/'
+# Open yaml for the default yaml and read what file to lookup for default menu
+default_menu_file = open(MENU_BASE_PATH+'default.yaml')
```

Comment:
Needs exception handling. 
Open yaml file using "with" keyword and keep it in the try except block

Reply:
default.yaml file is mandatory and part of build process. it will be available in all topologies